components/camel-spring-parent/camel-spring-rabbitmq/src/test/java/org/apache/camel/component/springrabbit/integration/RabbitMQProducerIT.java

```diff
@@ -40,6 +46,92 @@
 
 public class RabbitMQProducerIT extends RabbitMQITSupport {
 
+    @Disabled("Load test used to verify producer performance CAMEL-22278")
+    @Test
+    public void testProducerLoadAndReceive() throws Exception {
+        // --- 1. AMQP Setup ---
+        // Ensures the RabbitMQ queue, exchange, and binding exist before the test.
+        ConnectionFactory cf = context.getRegistry().lookupByNameAndType("myCF", ConnectionFactory.class);
+        Queue q = new Queue("myqueue");
+        TopicExchange t = new TopicExchange("foo");
+        AmqpAdmin admin = new RabbitAdmin(cf);
+        admin.declareQueue(q);
+        admin.declareExchange(t);
+        admin.declareBinding(BindingBuilder.bind(q).to(t).with("foo.bar.#"));
+
+        // --- 2. Test Configuration ---
+        final int numThreads = 10;
+        final int totalDurationSeconds = 10;
+        final int snapshotTimeSeconds = 5;
+
+        // --- 3. Concurrency Tools ---
+        ExecutorService executor = Executors.newFixedThreadPool(numThreads);
+        final AtomicInteger messageCounter = new AtomicInteger(0);
+        final AtomicBoolean running = new AtomicBoolean(true);
+
+        // --- 4. Define the Message Sending Task ---
+        Runnable senderTask = () -> {
+            while (running.get()) {
+                template.sendBody("direct:start", "Load test message #" + messageCounter.get());
+                messageCounter.incrementAndGet();
+            }
+        };
+
+        // --- 5. Start the Load Test ---
+        System.out.println(
+                String.format("Starting load test with %d threads for %d seconds...", numThreads, totalDurationSeconds));
+        for (int i = 0; i < numThreads; i++) {
+            executor.submit(senderTask);
+        }
+
+        // --- 6. Wait and Count at 5 Seconds ---
+        Thread.sleep(TimeUnit.SECONDS.toMillis(snapshotTimeSeconds));
+
+        int countAfter5Seconds = messageCounter.get();
+        System.out.println(String.format(">>> Messages sent after %d seconds: %d", snapshotTimeSeconds, countAfter5Seconds));
+        Assertions.assertTrue(countAfter5Seconds > 0, "No messages were sent after 5 seconds.");
+
+        // --- 7. Wait for the Remaining Time ---
+        Thread.sleep(TimeUnit.SECONDS.toMillis(totalDurationSeconds - snapshotTimeSeconds));
+
+        // --- 8. Stop Threads and Gather Final Sent Count ---
+        System.out.println("Stopping threads...");
+        running.set(false);
+        executor.shutdown();
+        executor.awaitTermination(5, TimeUnit.SECONDS);
+
+        int finalSentCount = messageCounter.get();
+        System.out.println(String.format(">>> Total messages sent in %d seconds: %d", totalDurationSeconds, finalSentCount));
```

Comment:
no system out in tests - use log.debug / log.info and so on

Reply:
ops, yeah, I was in hurry